### ISSUE TITLE
Config variable to expose the supervisor api on the network.

### ISSUE THREAD

**Issue by zvin** [resin-io/resin-supervisor#650]
Front conversation: https://app.frontapp.com/open/cnv_s2bgib

**Comment by shaunmulligan** [comment]
@zvin can we put a short description of the issue/feature in the github issue here since this is a public repo and most people don't have access to the links you posted

**Comment by zvin** [comment]
It's quite simple: if some en var is set (the name is yet to be defined), the supervisor will listen on 0.0.0.0 instead of 127.0.0.1 allowing people on the same network as the device to use its api. You will still require to pass a resin supervisor api key.

**Comment by pcarranzav** [comment]
+1 to this - and listening on the network should probably be the default setting in local mode and in offline mode, right @shaunmulligan?

(btw the supervisor does already listen on 0.0.0.0, what we have is iptables rules rejecting external connections other than through the resin-vpn interface)

**Comment by shaunmulligan** [comment]
@pcarranzav yeah I think it should probably be the default in localMode...not sure about offline mode as you could have a production version of an unmanaged OS that might not want this.
